### PR TITLE
Accessibility Pair Programming PR

### PR DESCRIPTION
**Pull request by TiteLunda**
This PR contains The correction made to improve the accessibility  of the webpage

1. Add a section Id to the navbar to help navigate into sections
2. Correction of the heading
3. Correction of the social media Using <ul>tag https://github.com/Brian3Kamau/Brian3Kamau.github.io/blob/ec971197fb083380049aea10a53ce9d4f7fcb373/index.html#L72-L78
4. Setting the position of the navbar to fixed,  at the top of the page. 

### Accessibility Report 

1. **Page titles:**  - [ ] No issues found

2.  **Image Text alternatives:**  - [ ] No issues found
3. **Text Headings:**  Corrections made to the headings. https://github.com/Brian3Kamau/Brian3Kamau.github.io/blob/4e86dde3b733870cb25a28933366504bc8bceec4/index.html#L62-L70
4. **Color Contrast:**
5. **Resize:** No issue found
6. **Interactions:** Labels with the hidden attribute added to the input tags to improve accessibility https://github.com/Brian3Kamau/Brian3Kamau.github.io/blob/ec971197fb083380049aea10a53ce9d4f7fcb373/index.html#L258-L266
7. **Moving Content:** No issues found 
8. **Multimedia:**  No issue found 
9. **The Basic structure of the page:** Navbar edited to improve the page structure https://github.com/Brian3Kamau/Brian3Kamau.github.io/blob/ec971197fb083380049aea10a53ce9d4f7fcb373/index.html#L32-L36

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,14 +39,18 @@
             alt="battery-icon"
             id="battery"
           />
-        </nav>  
+        </nav> 
+    
         <div class="desktop-only">
+          <nav class="desc-nav1"> 
           <ul class="desk-nav-link-ul">
```
Comment (TiteLunda) [inline index.html]: Setting the position of the navbar to fixed, at the top of the page.

File: index.html [hunk 2]
```diff
@@ -39,14 +39,18 @@
...
-            <li class="desk-nav-link" ><a href="#" class="desk-nav-link" id="desk-nav-link-1">Portfolio</a> </li>
-            <li class="desk-nav-link" ><a href="#" class="desk-nav-link" id="desk-nav-link-2"> About</a></li>
-            <li class="desk-nav-link" ><a href="#" class="desk-nav-link" id="desk-nav-link-3"> Contact</a></li>
+            <li class="desk-nav-link" ><a href="#top-section" class="desk-nav-link" id="desk-nav-link-1">Portfolio</a> </li>
+            <li class="desk-nav-link" ><a href="#works-section" class="desk-nav-link" id="desk-nav-link-2"> About</a></li>
+            <li class="desk-nav-link" ><a href="#contact-me" class="desk-nav-link" id="desk-nav-link-3"> Contact</a></li>
```
Comment (TiteLunda) [inline index.html:35]: Add of sections id to the navbar to help navigate through sections

File: index.html [hunk 3]
```diff
@@ -69,35 +73,26 @@ <h1 id="header-text">
...
-              ><img src="./images/github.png.jpeg" alt="github-icon" />
-            </a>
-            <a href="https://www.google.com"
-              ><img src="./images/fing.png.jpeg" alt="fng-icon" />
-            </a>
-          </div>
-        </div>
-      </div>
-    </section>
+          </h3>
```
Comment (TiteLunda) [inline index.html:70]: Add of a H3 header for a best accessibility and best practice

File: index.html [hunk 4]
```diff
@@ -69,35 +73,26 @@ <h1 id="header-text">
...
+    
+        <ul class="contact-icons">
+          <li> <a href="https://twitter.com/bk_rian"><img src="./images/twitter.svg" alt="twitter-icon"/></a></li>
+          <li>  <a href="https://www.linkedin.com/feed/"><img src="./images/in.png.jpeg" alt="linked-icon" /></a></li>
+          <li> <a href="https://github.com/Brian3Kamau"><img src="./images/M.png.jpeg" alt="Mail-icon" /></a></li>
+          <li><a href="https://github.com/Brian3Kamau"><img src="./images/github.png.jpeg" alt="github-icon" /></a></li> 
+          <li><a href="https://www.google.com"><img src="./images/fing.png.jpeg" alt="fng-icon"/></a></li> 
+        </ul>
+    </div>
```
Comment (TiteLunda) [inline index.html:79]: Use of <ul> tag for a best practice

File: style.css [hunk 5]
```diff
@@ -1059,6 +1065,15 @@ section {
...
   }
+  .desc-nav1{
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 0;
+    height: 48px;
+    position: fixed;
+    background-color: #fff;
+    z-index: 1;
+  }
```
Comment (TiteLunda) [inline style.css]: Add of new class for styling and making the navbar fixed

**Review by frank1738 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Documentation 👍 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**